tests/FileReaderTrait.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Behat Gherkin Parser.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Tests\Behat\Gherkin;
+
+use RuntimeException;
+
+trait FileReaderTrait
```

Comment:
I'd probably have made this a helper class with public static methods - curious about why you chose a trait here (I'm not saying it's wrong)?

Reply:
No particular reason.. even though traits theoretically make it more manageable.

I've also wondering introducing [thecodingmachine/safe](https://github.com/thecodingmachine/safe), at least for dev/testing... wdyt?

Reply:
I'm also fine with a static class approach, I'll go for that.

Reply:
I realised that we already have `symfony/filesystem` as a dev dependency, whereas `symfony/finder` is also included, but not as an explicit dependency, so might as well use both instead.

Reply:
Damnit... `symfony/filesystem` introduced `readFile` in v7.1, which supports PHP 8.2+, but we want to support PHP 8.1 as well. 😩